grails-app/assets/javascripts/pageAggregation/pageAggregationGuiHandling.js

```diff
@@ -234,10 +279,20 @@ OpenSpeedMonitor.ChartModules.GuiHandling.pageAggregation = (function () {
             queryData.toComparative = comparativeTimeFrame[1].toISOString();
         }
         spinner.start();
-        getDataForAggregationValue("median", queryData, isStateChange);
+
+        currentQuery = queryData;
+
+        getDataForAggregationValue(percentile, queryData, isStateChange);
         getDataForAggregationValue("avg", queryData, isStateChange);
     };
 
+    var loadPercentile = function (isStateChange) {
```

Comment:
Why do we need a isStateChange param, its not used and in jobGroupGuiHandling this function has no param